src/ml_board/frontend/Dashboard/src/components/settings/Settings.tsx

```diff
@@ -42,19 +46,21 @@ const Settings: React.FC<FuncProps> = (props) => {
                 setConfigTextState(settingConfigs);
             });
         }
-    },[])
+    },[isSocketConnected && (grid_search_id || rest_api_url || socket_connection_url)])
```

Comment:
`[isSocketConnected , grid_search_id , rest_api_url , socket_connection_url]` ?

Reply:
instead of the "OR ( || )" condition, it's a good practice to write like this. And while making changes I figured we didn't need "&" condition

Reply:
This part disappeared, I can see it in the **Files changed** section of the PR ?!